src/cba-list-new/cba-list-new.js

```diff
@@ -118,15 +126,40 @@ class List extends HTMLElement
     this.tooltipLinkText = this.getAttribute("tooltip-link-text");
     this.tooltipLinkTextDefault = "Learn more";
     this.connected = true;
+    this.group = this.shadowRoot.querySelector("#group");
 
     this.container.addEventListener("click", ({target}) =>
     {
-      if (target.tagName === "BUTTON")
+      switch (target.dataseet.action)
```

Comment:
It's `dataset` probably TYPO breaks functionality here.

Reply:
You also need to add `data-action` to the elements you expect to provide the actions.